addons/print/data/set_default_printer.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<odoo>
+  <data noupdate="1">
```

Comment:
Don't think `noupdate` is needed here, the `yml` file didn't set it (not sure if it can anyway) and not sure `noupdate` even applies to function calls.